python/lsst/ts/observing/utilities/auxtel/latiss/utils.py

```diff
@@ -0,0 +1,26 @@
+__all__ = ["parse_obs_id"]
+from lsst.ts.observatory.control.constants import atcs_constants
+
+
+def parse_obs_id(obs_id):
```

Comment:
Is this really AT specific? I think we could make it generic enough to parse nobs_id in general, no?

Also, consider using regular expression:

```
import re

obs_id_pattern = re.compile(r"(?P<source>[a-zA-Z][a-zA-Z])_(?P<controller>[a-zA-Z])_(?P<day_obs>[0-9]*)_(?P<seq_num>[0-9]*)")

group = obs_id_pattern.match("AT_O_20200219_000212")

group.group("source"),group.group("controller"),group.group("day_obs"),group.group("seq_num")
```